src/scss/shared/_mixins.scss

```diff
@@ -1,12 +1,3 @@
-/// Alert message content highlight
-///
-/// Outputs styles for highligted message content
-@mixin oMessageContentHighlight {
```

Comment:
This needs to hang around to not be breaking. We could warn that it's deprecated but it might be better to keep with just `@include oTypographyBold('sans')`, then we can easily modify what highlighted content looks like in the future.

Reply:
:retreat: